### ISSUE TITLE
Thunderbird Accordion Will not open

### ISSUE THREAD

**Issue by tankerkiller125** [mjmlio/mjml#1670]
**Describe the bug**
The MJML Accordion feature does not work properly in Thunderbird, in outlook the accordion is automatically expanded which seems to work properly and it also seems to work well in Gmail. However Thunderbird does not automatically expand the accordion and also does not allow the accordion to expand.

**To Reproduce**
1. Install the latest version of thunderbird
2. Create an MJML template using an accordion in the https://mjml.io/try-it-live/ area
3. Copy the HTML from the try it live page into a new email and send it to the email address Thunderbird is attached to
4. Attempt to expand/view accordion (will not work)

**Expected behavior**
The accordion should automatically expand in Thunderbird or it should allow expansion

**MJML environment (please complete the following information):**
 - OS: [e.g. MacOS] Windows
 - MJML Version [e.g. 4.2.0] Unknown
 - MJML tool used [e.g MJML App] Try It Live Page

**Email sending environment(for rendering issues)**:
 - Platform used to send the email [e.g [Putsmail](https://putsmail.com/)]  

**Affected email clients (for rendering issues):**
 - Email Client [e.g Gmail] Thunderbird
 - OS: [e.g. Windows] Windows
 - Browser [e.g. Google Chrome] NA

**Comment by iRyusa** [comment]
I don't think this was really tested on thunderbird anyway. https://www.htmlemailcheck.com/knowledge-base/target-mozilla-thunderbird-using-moz-text-html-css-class/ Maybe we can add a specific attribute to target it, doesn't seem too bad to add but it would need some test anyway

**Comment by iRyusa** [comment]
If anyone willing to contribute on that issue it should be an easy first contribution with the provided link !

**Comment by srkyaganti** [comment]
Hi @iRyusa,
Is this issue still open, or is anyone working on it. If no, I'd like to take it up

**Comment by kmcb777** [comment]
Hi @srkyaganti no one is working on this right now, your contribution would be welcome !

**Comment by srkyaganti** [comment]
Hi @kmcb777 , Can someone please assign this issue to me so that it is not marked as open for contribution.

Thanks,
Srikar

**Comment by kmcb777** [comment]
done 👍

**Comment by bertho-zero** [comment]
My workaround:

```html
    <mj-style>
      .moz-text-html input.mj-accordion-checkbox + * .mj-accordion-title {
        cursor: auto;
        touch-action: auto;
        -webkit-user-select: auto;
        -moz-user-select: auto;
        user-select: auto;
      }

      .moz-text-html input.mj-accordion-checkbox + * .mj-accordion-content {
        overflow: hidden;
        display: block;
      }

      .moz-text-html input.mj-accordion-checkbox + * .mj-accordion-ico {
        display: none !important;
      }
    </mj-style>
```

**Comment by iRyusa** [comment]
That seems great can you open a PR on that ?

**Comment by bertho-zero** [comment]
Done in https://github.com/mjmlio/mjml/pull/1846

**Comment by iRyusa** [comment]
We'll try to review that asap 👍 thanks !

**Comment by iRyusa** [comment]
I think this has been released in 4.6.X so we can close it!
